baseplate/observers/metrics_tagged.py

```diff
@@ -52,17 +45,16 @@ def from_config_and_client(
             raw_config,
             {
                 "metrics": {
-                    "whitelist": config.Optional(
-                        config.TupleOf(config.String),
-                        default=["client", "endpoint", "success", "error"],
+                    "allowlist": config.Optional(
+                        config.TupleOf(config.String), default=["client", "endpoint"],
```

Comment:
I think this approach makes it possible for service owner to remove `client` and/or `endpoint` from the allowlist (either accidentally or intentionally), thus making the metrics no longer comply to the baseplate spec. should we hardcode these 2 to be always included instead?

technically with the new code we are actually hardcoding `success` tag,

Reply:
:+1: makes sense. i've added those as always allowed.